content/SCALE/SCALEUIReference/SystemSettings/Services/FTPServiceScreenSCALE.md

```diff
@@ -31,80 +31,79 @@ To configure FTP, go to **System Settings > Services** and find **FTP**, then cl
-| **Timeout** | Enter the maximum client idle time in seconds before disconnect. Default value is **600** seconds. |
+| **Login Attempts** | Enter the maximum attempts before the client disconnects. Increase if users are prone to misspellings or typos. |
+| **Notransfer Timeout** | Enter the maximum number of seconds a client is allowed to spend connected, after authentication, without issuing a command which results in creating an active or passive data connection (sending/receiving a file or receiving a directory listing). |
+| **Timeout** | Enter the maximum client idle time in seconds before disconnecting. The default value is **600** seconds. |
 
 ## FTP Advanced Settings
-**Advanced Settings** include the **General Options** on the **Basic Settings** configuration screen, and allow you to specify access permissions, TLS settings, bandwidth and other setting to further customize FTP access.
+**Advanced Settings** include the **General Options** on the **Basic Settings** configuration screen and allow you to specify access permissions, TLS settings, bandwidth, and other settings to customize FTP access.
 
 ### Access and TLS Setting Options
 
 ![FTPAdvancedSettingsAccess](/images/SCALE/22.12/FTPAdvancedSettingsAccess.png "Services FTP Advanced Settings Access Options")
 
 #### Access Settings 
-**Access** settings specify user login, file and directory access permissions.
+**Access** settings specify user login, file, and directory access permissions.
 
 | Settings | Description |
 |----------|-------------|
-| **Always Chroot** | Select to only allow users access their home directory if they are in the **wheel** group. This option increases security risk. To confine FTP sessions to a home directory of a local user, enable **chroot** and select **Allow Local User Login**. |
-| **Allow Root Login** | Select to allow root logins. This option increases security risk so enabling this is discouraged. Do *not* allow anonymous or root access unless it is necessary. 
-For better security, enable TLS when possible (especially when exposing FTP to a WAN). TLS effectively makes this [FTPS](https://tools.ietf.org/html/rfc4217). |
-| **Allow Anonymous Login** | Select to allow anonymous FTP logins with access to the directory specified in **Path**. Selecting this displays the **Path** field. Enter or browse to the loction to populate the field. |
-| **Allow Local User Login** | Select to allow any local user to log in. By default, only members of the **ftp** group are allowed to log in. |
+| **Always Chroot** | Only allows users to access their home directory if they are in the **wheel** group. This option increases security risk. To confine FTP sessions to a local user home directory, enable **chroot** and select **Allow Local User Login**. |
+| **Allow Root Login** | Select to allow root logins. This option increases security risk, so enabling this is discouraged. Do *not* allow anonymous or root access unless it is necessary. 
```

Comment:
is this one area where we need to comment on the admin user? did the UI end up with a new admin option? if not, we need to tell them how admin user uses this option.

Reply:
As far as I can tell, the Admin user works just like any other user as far as FTP is concerned.